packages/n4s/src/rules/endsWith/index.js

```diff
@@ -0,0 +1,7 @@
+function endsWith(value, arg1) {
+    return typeof value === 'string' && typeof arg1 === 'string' && value.endsWith(arg1);
+}
+
+endsWith.negativeForm = 'isNotEndsWith';
```

Comment:
I think that `doesNotEndWith` works better here

`enforce value does not end with`

```suggestion
endsWith.negativeForm = 'doesNotEndWith';
```